### PR TITLE
Update documentation of forwarding rule for vpc psc endpoint

### PR DESCRIPTION
**Pull request by IIBenII**
Fix this issue: https://github.com/hashicorp/terraform-provider-google/issues/12326

If this PR is for Terraform, I acknowledge that I have:

- [X] Searched through the [issue tracker](https://github.com/hashicorp/terraform-provider-google/issues) for an open issue that this either resolves or contributes to, commented on it to claim it, and written "fixes {url}" or "part of {url}" in this PR description. If there were no relevant open issues, I opened one and commented that I would like to work on it (not necessary for very small changes).
- [ ] [Generated Terraform](https://github.com/GoogleCloudPlatform/magic-modules#generating-the-terraform-providers), and ran `make test` and `make lint` to ensure it passes unit and linter tests.
- [ ] Ensured that all new fields I added that can be set by a user appear in at least one [example](https://github.com/GoogleCloudPlatform/magic-modules/tree/main/mmv1/templates/terraform/examples) (for generated resources) or [third_party test](https://github.com/GoogleCloudPlatform/magic-modules/tree/main/mmv1/third_party/terraform/tests) (for handwritten resources or update tests).
- [ ] [Ran](https://github.com/hashicorp/terraform-provider-google/blob/main/.github/CONTRIBUTING.md#tests) relevant acceptance tests (If the acceptance tests do not yet pass or you are unable to run them, please let your reviewer know).
- [ ] Read the [Release Notes Guide](https://github.com/GoogleCloudPlatform/magic-modules/blob/main/.ci/RELEASE_NOTES_GUIDE.md) before writing my release note below.

**Release Note Template for Downstream PRs (will be copied)**

```release-note:none

```

### CODE REVIEW

File: mmv1/templates/terraform/examples/forwarding_rule_vpc_psc.tf.erb
Comment (c2thorn) [inline mmv1/templates/terraform/examples/forwarding_rule_vpc_psc.tf.erb:7]: Will this value work in a test as-is? Or does it actually need to be set to a valid target to pass (accepted by the API and "created" but not necessarily functional)

**Review by c2thorn — CHANGES_REQUESTED** [review]

File: mmv1/templates/terraform/examples/forwarding_rule_vpc_psc.tf.erb
Comment (IIBenII) [reply mmv1/templates/terraform/examples/forwarding_rule_vpc_psc.tf.erb:7]: It's need to be set to a valid value, "as-is" will failed.

Comment (c2thorn) [reply mmv1/templates/terraform/examples/forwarding_rule_vpc_psc.tf.erb:7]: For our examples we like to have them testable and tests self-contained when possible. Can the target be set up in this example itself?

Comment (IIBenII) [reply mmv1/templates/terraform/examples/forwarding_rule_vpc_psc.tf.erb:7]: Ok, I will work on it

**Comment by IIBenII** [comment]
@c2thorn I fix all your feedback, thanks.

**Comment by slevenick** [comment]
Looks like the test fails during import as the returned value is the IP address not the name of the resource. Can you add an ignore import value on that?

**Comment by IIBenII** [comment]
@slevenick Everything look good?

**Review by slevenick — APPROVED** [review]

**Comment by slevenick** [comment]
Huh, unfortunately I can't merge as @c2thorn requested changes. May need to wait for a re-review from them before this can go in

**Comment by IIBenII** [comment]
@slevenick any news on it?

**Review by c2thorn — APPROVED** [review]

**Comment by wyardley** [comment]
`load_balancing_scheme` itself still doesn't seem to be documented? Created #22098 for that

**Pull request merged**
